python/ray/data/iterator.py

```diff
@@ -647,9 +654,11 @@ def make_generator():
                         key: convert_pandas_to_torch_tensor(
                             batch,
                             feature_columns[key],
-                            feature_column_dtypes[key]
-                            if isinstance(feature_column_dtypes, dict)
-                            else feature_column_dtypes,
+                            (
```

Comment:
is this just a lint change?

Reply:
My auto-linting caught this for some reason.